test/e2e/specs/editor/various/validate-multiple-use.spec.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Validate multiple use', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should display correct amount of warning message', async ( {
+		editor,
+		page,
+	} ) => {
+		// Insert a block with `multiple` feature enabled, such as `core/more`
+		await editor.insertBlock( {
+			name: 'core/more',
+		} );
+
+		const optionButton = page.locator(
+			".components-dropdown-menu__toggle[data-toolbar-item='true'][aria-label='Options']"
+		);
```

Comment:
We should aim to use `getByRole`. Why? Because it encourages us to access elements using accessible methods. This tends to better mimic things that are _perceivable_ to all types of users and helps to avoid us relying on developer APIs (such as classnames...etc). This helps us to create more accessible software as any changes made to the components under test that make them less perceivable are caught by the tests.

Reply:
Thanks @getdave for the insights, all locators now updated to `getByRole`.